### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by dutrieuc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# toggl_to_odoo
+
+toggle_to_odoo is a tool to synchronize timesheets between an Odoo database and Toggl Track
```
Comment (andreabak) [inline README.md]:
```suggestion
**toggle_to_odoo** is a tool to synchronize entries from **Toggl Track** to timesheets in an **Odoo** database
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Guide
+
+### Use
```
Comment (andreabak) [inline README.md]:
```suggestion
### Usage
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+To run the synchronization, run :
+```sh
+python3 -m toggl_to_odoo upload toggl2odoo https://www.odoo.com openerp history
+```
```
Comment (andreabak) [inline README.md]:
````suggestion
To run the synchronization, run the following command from within the repo root directory, with the appropriate arguments for your setup:
```sh
python3 -m toggl_to_odoo upload <converter_name> <odoo_url> <odoo_db_name> <history_file_path>
```
````

File: README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+## Setup
+###### Dependancies
+- [toggl Cli](https://toggl.uhlir.dev/)
```
Comment (andreabak) [inline README.md]:
````suggestion
Set up the dependencies for your python3 environment:
```sh
pip3 install -r requirements.txt
```
````

File: README.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+###### Install
+
+- [Setup Toggl CLI](https://toggl.uhlir.dev/#configuration)
+- Clone this repository
```
Comment (andreabak) [inline README.md]:
````suggestion
- Clone this repository
- Install dependencies for your python3 environment:
   ```sh
   pip3 install -r requirements.txt
   ```
- [Setup Toggl CLI config](https://toggl.uhlir.dev/#configuration)
````

**Review by andreabak — APPROVED** [review]
I've added some suggestion for slightly different wording.
The sections are a bit backwards imo, like I would put `Setup` before `Usage` and remove the dependency section altogether.
As for the last section about converters, I would move it to a README inside the `converters` package and only for the `odoo-abk-converters` branch, but it can be done at a later time.

File: README.md (hunk 3 above)
Comment (dutrieuc) [reply README.md]: Of course people should adapt the line to their setup but maybe having an idea of a reasonable default would be good ?
It took me a little bit of time to figure out what were the available converter names and about the history thing

Comment (andreabak) [reply README.md]: I think having a cli command that lists the available converters is a better solution. For the rest of the command line it should already be documented with `--help`.
About converters I'm not sure if there should be defaults 🤔, the ones I wrote are tailored to my entries naming scheme etc., and other users will want to adapt them for their setup. Also I'd like to keep this tool more general than for only our `odoo.com` usecase, or at least in its master branch.
Maybe some very generic ones could be written, eg. some that take a "standard" toggl entry naming format and derive task id (I think project can then be derived from task) and the description.

**Pull request merged**